docs/installation.rst

```diff
@@ -28,10 +28,7 @@ Required
 * Python >= 3.9 (3.9 required for entry points)
 
   * included in `Complete Local conda-based GeoIPS Installation`_
-* cartopy >= 0.20.3 (0.21.0 required for matplotlib 3.6.0, does not install
-  cleanly via pip)
-
-  * included in `Complete Local conda-based GeoIPS Installation`_
+* libgeos-dev (required to install cartopy via pip)
```

Comment:
I think this should still have the "included in 'complete local conda-based geoips installation" sub bullet (since you do not have to worry about libgeos-dev if you are installing via conda)